DOCS-3077, DOCS-3037, DOCS-3038 AppSec Landing Page, Getting Started, Setup n Config (#12951)
DataDog/documentation: layouts/shortcodes/appsec-getstarted.md

In general, setting up Application Security monitoring involves:

> 1. Installing the Datadog library (the most recent APM tracing library) and enabling it within your services.

Author: ```suggestion
1. Updating to the latest Datadog library (the most recent APM tracing library).
```
